flink-runtime/src/test/java/org/apache/flink/runtime/scheduler/strategy/LazyFromSourcesSchedulingStrategyTest.java

```diff
@@ -377,4 +349,20 @@ private LazyFromSourcesSchedulingStrategy startScheduling(TestingSchedulingTopol
 		schedulingStrategy.startScheduling();
 		return schedulingStrategy;
 	}
+
+	private void checkLatestScheduledVertices(final List<TestingSchedulingExecutionVertex> expected) {
```

Comment:
I think that by only looking at the method name it is difficult to understand what `checkLatestScheduledVertices` does, i.e., is the parameter the expected result or the actual result that will be checked. I would propose the following name: `assertLatestScheduledVerticesAreEqualTo`

Reply:
Yes. That's better. updated.